spec/support/examples_group/shared_examples_for_ownership_mixin.rb

```diff
@@ -2,23 +2,60 @@
   context "includes OwnershipMixin" do
     include Spec::Support::ArelHelper
 
-    let(:user) { User.where(:userid => "ownership_user").first }
-
-    before do
-      user = FactoryGirl.create(:user,
-                                :userid     => "ownership_user",
-                                :miq_groups => FactoryGirl.create_list(:miq_group, 1))
-      user2 = FactoryGirl.create(:user)
-
-      factory = described_class.to_s.underscore.to_sym
-      FactoryGirl.create(factory, :name => "in_ldap",     :miq_group_id => user.current_group.id)
-      FactoryGirl.create(factory, :name => "not_in_ldap", :miq_group => FactoryGirl.create(:miq_group))
-      FactoryGirl.create(factory, :name => "no_group")
-      FactoryGirl.create(factory, :name => "user_owned",  :evm_owner => user)
-      FactoryGirl.create(factory, :name => "user_owned2", :evm_owner => user2)
+    let(:user) do
+      FactoryGirl.create(:user,
+                         :userid     => "ownership_user",
+                         :miq_groups => FactoryGirl.create_list(:miq_group, 1))
     end
 
-    describe ".owning_ldap_group" do
+    let(:user2)  { FactoryGirl.create(:user) }
+    let(:group)  { user.current_group }
+    let(:group2) { FactoryGirl.create(:miq_group) }
+
+    let(:factory) { described_class.to_s.underscore.to_sym }
```

Comment:
> Indifferent. I look for what factory refers to and find this pretty easily

@chrisarcand it's more about convention.  Is `described_factory` a useful convention?  I don't know.